guides/v2.2/cloud/project/profile-database-queries.md

```diff
@@ -0,0 +1,69 @@
+---
+group: cloud-guide
+title: Profile database queries
+functional_areas:
+  - Cloud
+  - Setup
+  - Services
+---
+
+When making changes to a database, you may find it helpful to run profiling to identify the impact of those changes. Profiling maintains a log of database queries and increases run time precision.
```

Comment:
When "run time" is used as an adjective, shouldn't it be hyphenated?

Reply:
yes. i'm getting rusty